src/validateRegisterForm.test.js

```diff
@@ -3,28 +3,145 @@
 describe(`Function 'validateRegisterForm':`, () => {
   const validateRegisterForm = require('./validateRegisterForm');
 
-  it(`should be declared`, () => {
-    expect(validateRegisterForm).toBeInstanceOf(Function);
-  });
+  describe(`Generic tests`, () => {
```

Comment:
Generic tests - too abstract name
Why do we use here `` instead of ''?  - please fix it everywhere. `` - for interpolation